server/controllers/nc_news_controller.js

```diff
@@ -9,3 +10,17 @@ exports.getAllTopics = (req, res, next) => {
       next(err); //Pass errors to error-handling middleware
     });
 };
+
+exports.getArticleById = (req, res, next) => {
+  const { article_id } = req.params;
+  if (isNaN(article_id)) {
+    return res.status(400).send({ msg: "Bad Request" });
+  }
```

Comment:
We should not be handling our errors manually like this. We should firstly do this kind of error handling in the model by rejecting the promise so it enters your .catch in this controller file and will be handled by your middleware error handlers.